matrix/docker/matrix_converter.py

```diff
@@ -165,92 +178,93 @@ def _to_loom(self):
            output_path: Path to the new loom file.
         """
 
+        def _timestamp():
```

Comment:
`matrix.common.date.get_datetime_now` does this (with a slightly different format string)